Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1569,6 +1575,8 @@ def gen2018HiMix(fragment,howMuch):
 steps['RECOHI2015']=merge([hiDefaults2015,{'-s':'RAW2DIGI,L1Reco,RECO,VALIDATION,DQM'},step3Up2015Defaults])
 steps['RECOHI2011']=merge([hiDefaults2011,{'-s':'RAW2DIGI,L1Reco,RECO,VALIDATION,DQM'},step3Defaults])
 
+steps['RECOPPREF2017']=merge([ppRefDefaults2017,{'-s':'RAW2DIGI,L1Reco,RECO,EI,PAT,VALIDATION:@standardValidation+@miniAODValidation,DQM:@standardDQM+@miniAODDQM'},step3Up2015Defaults])
```

Comment:
if all steps are the same as in step3Up2015Defaults, why does this need a copy-paste here?

Reply:
it looks like RECOSIM step was dropped.
Was it intentional?
This step is new starting from 940pre3 and is expected to be operational for all MC workflows.

Reply:
Changing that line to the following should address these two comments.
steps['RECOPPREF2017']=merge([ppRefDefaults2017,step3Up2015Defaults])
Can you confirm ?

Reply:
yes, visually [assuming python result works out], that's what I expect.

Reply:
The wf worked again after this change. 
I assume the following line also needs to be changed correspondingly. It is for xexe wf, the one that starts with steps['RECOHI2017']
https://github.com/ttrk/cmssw/blob/f63d6831b0de5f50177957bdf17b1053d9208f17/Configuration/PyReleaseValidation/python/relval_steps.py#L1574

Reply:
sure, although this may be a bit outside of this PR context

Reply:
Then I will not touch that other line, make the change only for this PR's era.

Reply:
> Then I will not touch that other line, make the change only for this PR's era.

is the mod for RECOPPREF2017 coming soon?
Please clarify.

Reply:
I just committed the change.